### PR TITLE
Turtle writer options

### PR DESCRIPTION
**Pull request by afs**

### CODE REVIEW

File: source/documentation/io/rdf-output.md [hunk 1]
```diff
@@ -240,7 +241,58 @@ but always writes one complete triple on one line (no use of `;`).
 | TURTLE_FLAT |
 | TRIG_FLAT   |
 
-### N-Triples and N-Quads
+### Turtle and Trig format options {#opt-turtle-trig}
+
+Some context settings affect the output of Turtle and TriG writers. Unless
+otherwise noted, the settign applies to both Turtle and TriG.
```
Comment (kinow) [inline source/documentation/io/rdf-output.md:247]: s/the settign/the setting

File: source/documentation/io/rdf-output.md [hunk 2]
```diff
@@ -240,7 +241,58 @@ but always writes one complete triple on one line (no use of `;`).
...
+
+| Context setting | Cmd line | Values |
+|-----------------|----------|--|
+| RIOT.symTurtleDirectiveStyle | "ttl:directiveStyle" | "sparql", "rdf11", "at", "n3" |
+| RIOT.symTurtleOmitBase       | "ttl:omitBase"       | "true", "false" |
+
+<p>&nbsp;</p>
```
Comment (kinow) [inline source/documentation/io/rdf-output.md:254]: Ha, nice trick!

File: source/documentation/io/rdf-output.md [hunk 3]
```diff
@@ -240,7 +241,58 @@ but always writes one complete triple on one line (no use of `;`).
...
+  RDFWriter.create()
+     .set(RIOT.symTurtleDirectiveStyle, "sparql")
+     .lang(Lang.TTL)
+     .source(model)
+     .output(System.out);
+```
+##### _Base URI_
+
+Output can be written with relative URIs and no base. Note: such output is not
+portable; it's meaning depends on the base URI at the time of reading.
```
Comment (kinow) [inline source/documentation/io/rdf-output.md:281]: s/it's/its

**Review by kinow — APPROVED** [review]
Couple comments, no blockers :+1:

File: source/documentation/io/rdf-output.md (hunk 2 above)
Comment (afs) [reply source/documentation/io/rdf-output.md:254]: Somewhat crude really! But to tweak the CSS, and then have it ripple out across the site is a somewhat bigger task.

**Review by rvesse — APPROVED** [review]

**Comment by kinow** [comment]
Are we waiting for a PR in Jena before merging this one @afs ?

**Comment by afs** [comment]
This is for 3.17.0 - I'll note when things need to wait for a release next time.

https://issues.apache.org/jira/browse/JENA-1968

**Comment by afs** [comment]
Applied locally from diff, edited to resolve the merge conflict.

**Pull request closed without merging**